hubblestack_nova_profiles/cis/ubuntu-1204-level-1-scored-v1-0-0.yaml

```diff
@@ -221,7 +221,7 @@ grep:
       data:
         Ubuntu-12.04:
         - /etc/ssh/sshd_config:
-            pattern: X11Forwarding
+            pattern: XForwarding
```

Comment:
Also confused in the workflow - this is supposed to be orthogonal to #49, and I thought I rebased before editing this one but somehow the two appear bundled.